examples/ssr/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import "fast-todo-app";
+import { app, DefaultTodoList, TodoList } from "fast-todo-app";
+
+fetch("http://localhost:8080/todos")
```

Comment:
Instead of doing this fetch, I think the server some write this json into a known location somewhere, probably vis a script tag and then this code should read that out and pass it to the DefaultTodoList. That prevents the extra network request which shouldn't be necessary if the server already has the data. Just send it down as part of the payload.

Reply:
Yeah we can do that, I'm not quite following the implementation you're proposing though. Embed the JSON data as JavaScript into a script tag that exposes it as a global var to be used by bundle.js? The only other option I can think is to store it as an attr somewhere

Reply:
Seems the global var is what Redux is proposing: https://redux.js.org/usage/server-rendering#inject-initial-component-html-and-state

Reply:
Cool, working on it now. I found a bug in SSR trying to implement this that I need to track down before I push this up.